Tests/Functional/VictoireWebTestCase.php

```diff
@@ -0,0 +1,181 @@
+
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    /**
+     * @var ManagerRegistry
+     */
+    protected $doctrine;
+
+    /**
+     * @var ObjectManager
+     */
+    protected $entityManager;
+
+    /**
+     * Start tests by set up Kernel, Doctrine and Entity Manager.
+     */
+    protected function setUp()
+    {
+        $this->setUpSymfonyKernel();
+        $this->setUpDoctrine();
+    }
+
+    /**
+     * Drop and create database schema.
+     *
+     * @throws \Doctrine\ORM\Tools\ToolsException
+     */
+    protected function resetSchema()
+    {
+        if ($metadata = $this->getMetadata()) {
+            $schemaTool = new SchemaTool($this->entityManager);
+            $schemaTool->dropSchema($metadata);
+            $schemaTool->createSchema($metadata);
+        }
+    }
+
+    /**
+     * Victoire login.
+     *
+     * @param Client $client
+     * @param array  $roles
+     */
+    protected function logIn(Client $client, array $roles)
+    {
+        $session = $client->getContainer()->get('session');
+
+        $token = new UsernamePasswordToken('test', null, self::VICTOIRE_FIREWALL, $roles);
+        $session->set('_security_'.self::VICTOIRE_FIREWALL, serialize($token));
+        $session->save();
+
+        $cookie = new Cookie($session->getName(), $session->getId());
+        $client->getCookieJar()->set($cookie);
+    }
+
+    /**
+     * Load seeds fixtures and given fixtures.
+     *
+     * @param array $fixtures
+     */
+    protected function loadFixtures(array $fixtures = [])
+    {
+        $fixtures = array_merge($fixtures, $this->getSeeds());
+        $faker = new \Faker\Generator();
+        Fixtures::load(
+            $fixtures,
+            $this->entityManager,
+            [
+                'providers' => [
+                    $this,
+                    new \Faker\Provider\Base($faker),
+                ],
+            ]
+        );
+        $this->resetViewsReference();
+    }
+
+    /**
+     * Reset Victoire ViewReferences.
+     */
+    protected function resetViewsReference()
+    {
+        $viewsReferences = $this->container->get('victoire_core.view_helper')->buildViewsReferences();
+        $this->container->get('victoire_view_reference.manager')->saveReferences($viewsReferences);
+    }
+
+    /**
+     * Set up Symfony Kernel and provide container.
+     */
+    private function setUpSymfonyKernel()
+    {
+        static::$kernel = $this->createKernel();
+        static::$kernel->boot();
+
+        $this->container = static::$kernel->getContainer();
+    }
+
+    /**
+     * Provide doctrine and entity manager.
+     */
+    private function setUpDoctrine()
+    {
+        $this->doctrine = $this->createDoctrineRegistry();
+        $this->entityManager = $this->doctrine->getManager();
+    }
+
+    /**
+     * Get seeds files for Users, Pages and Templates.
+     *
+     * @return array
+     */
+    private function getSeeds()
+    {
+        return [
```

Comment:
I tried to do that but seeds must be loaded in a specific order due to some relations between entities. For example here we have to load `template.yml` before `page.yml`. So it's better to do it manually.